edge-apps/tfl-bus-status/screenly.yml

```diff
@@ -0,0 +1,19 @@
+---
+app_id: 01HXF4KGCCWNAK1TQRBTTS4X47
+description: Displays TFL Bus Status.
+con: 'https://playground.srly.io/edge-apps/tfl-bus-status/static/images/icon.svg'
+author: Screenly, Inc.
+entrypoint: index.html
+settings:
+  Stop_ID:
+    type: string
+    default_value: 490006428B
+    title: Bus Stop ID
+    optional: false
+    help_text: TFL Bus Stop ID.
+  TFL_API:
```

Comment:
The capitalization here does not align with capitalization for other Edge Apps. Let's make it consistent.

Reply:
Noted. Thanks